### DISCUSSION TITLE
Livewire 3 - navigate. Make re-load stylesheet

### DISCUSSION THREAD

**Discussion by TitanDvd** [livewire/livewire#6037] [3. Feature ideas]
I have two pages. Home and login. Those pages have its own stylesheet each one. When user `wire:navigate`'s from home to login page, styles from home persists in login. I suggest some kind of optional mechanism to prevent this from happening. 
For example: `<link rel = " . . . " href = " . . . " wire:navigate.goaway>`

**Comment by danie-ramdhani** [comment]
you can use blade directive `@if` for that or use different layout. here's the link: https://livewire.laravel.com/docs/components#per-component-layout-configuration

**Comment by TitanDvd** [comment]
In a world with a fully loaded page, it works. But, with the navigation feature in Livewire 3, `<head>` only loads once, so it leaves what was in the head and if there is something new in the new load, it puts it leaving the old one. In fact, if you move the `link` tag to the html body, it works perfectly, ie different stylesheets are loaded in each `wire:navigate`. I am using this way to load different layouts.
```
 return view('livewire.common.register-component')
                ->extends('components.auth_register_layout')
                    ->section('content');
```
